src/themes/default/components/core/blocks/Header/MicrocartIcon.vue

```diff
@@ -28,4 +24,20 @@ export default {
   .microcart-icon {
     display: inline-flex;
   }
+
+  .minicart-count {
+    position: relative;
+    display: flex;
+    top: -5px;
+    left: -5px;
+    width: 16px;
+    height: 16px;
+    border-radius: 50%;
+    justify-content: center;
+    align-items: center;
+    font-size: 10px;
```

Comment:
please use atomic classes for font-size. Look at [text.css](https://github.com/DivanteLtd/vue-storefront/blob/master/src/themes/default/css/text.css) and extend it so it can meet your requirements ;)